test/data/legalDeclationHTML.html

```diff
@@ -37,117 +37,127 @@
 <div id="global-header-bar"></div>
 <main id="content" role="main">
     <div class="phase-banner-beta">
-        <p>
+        <p class="govuk-body">
             <strong class="phase-tag">Beta</strong>
             <span>This is a new service – your <a aria-label="Feedback: this link will open in a new tab. You&rsquo;ll need to return to this tab to continue with your application." href="https://www.smartsurvey.co.uk/?pageurl=/declaration" target="_blank" rel="external noopener">feedback</a> will help us to improve it.</span>
         </p>
     </div>
-    <div class="grid-row">
-        <div class="column-two-thirds">
-            <h1 class="form-title heading-large">Check the legal statement and make your declaration</h1>
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <h1 class="form-title govuk-heading-l">Check the legal statement and make your declaration</h1>
             <form class="form" method="post" action="/declaration" novalidate>
                 <input type="hidden" name="_csrf" value="2z1FpoVq-4sBP_Td3IN7EOL029ZDofC_HAH0">
-                <p>This statement is based on the information you&rsquo;ve given in your application. It will be stored as a public record.</p>
+                <p class="govuk-body">This statement is based on the information you&rsquo;ve given in your application. It will be stored as a public record.</p>
                 <div class="declaration-wrapper">
                     <div class="inset-page declaration-grid-item">
-                        <div class="heading-medium declaration-header">In the High Court of Justice</div>
-                        <div class="heading-small declaration-header">Family Division</div>
-                        <div class="heading-small declaration-header">(Probate)</div>
-                        <h2 class="heading-medium declaration-subheader">Legal statement</h2>
-                        <p class="declaration-item">I, Jason Smith of An address
+                        <div class="declaration-header">
+                            <div class="govuk-heading-m declaration-header-item">In the High Court of Justice</div>
```

Comment:
should probably be using h3, h4 tags for accessibility

Reply:
Doesn't really matter, this is just for a test...

Reply:
how is this file used for test? should you not be loading actual file for test?